### PR TITLE
feat: add disclaimer and use "built on KILT"

### PR DESCRIPTION
**Pull request by tjwelde**
This PR adds a link to the kilt.io disclaimer and renames showcase to "built on KILT"

### CODE REVIEW

File: docs/develop/05_builtonkilt.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+title: Built on KILT
 ---
 
 import ThemedImage from '@theme/ThemedImage';
 import LogoText from '@site/src/components/LogoText';
 
-This section contains a non-exhaustive list of apps, wallets and websites that integrate KILT.
-If you are missing from the list and want to be added, you are welcome to [open a PR](https://github.com/KILTprotocol/docs/edit/develop/docs/develop/showcase.md).
+This section contains a non-exhaustive list of apps, wallets and websites that are built on KILT.
+If you are missing from the list and want to be added, you are welcome to [open a PR](https://github.com/KILTprotocol/docs/edit/develop/docs/develop/05_builtonkilt.md).
```
Comment (ntn-x2) [inline docs/develop/05_builtonkilt.md:10]: This link seems to be broken.

Comment (tjwelde) [reply docs/develop/05_builtonkilt.md:10]: yes, until this is merged it will be

Comment (ntn-x2) [reply docs/develop/05_builtonkilt.md:10]: I think it does not need the number in front as the ID does not have it.

File: src/pages/index.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ const featureRows = [
       link: 'docs/concepts/what-is-kilt',
       description: (
         <>
-          Learn about KILT identity components and why they matter.
-          Read on KILT credentials and what they provide.
+          Learn about KILT identity components and why they matter. Read on KILT
```
Comment (ntn-x2) [inline src/pages/index.js:18]: One line one sentence?

File: docs/develop/05_builtonkilt.md (hunk 1 above)
Comment (ntn-x2) [reply docs/develop/05_builtonkilt.md:10]: All good then, you're right.

File: src/pages/index.js (hunk 2 above)
Comment (tjwelde) [reply src/pages/index.js:18]: how did this happen? Must be the auto formatter or something

**Review by ntn-x2 — APPROVED** [review]
👌

**Pull request merged**